templates/initial-setup/connection_profile.php

```diff
@@ -125,51 +150,5 @@
       </form>
     </div>
 
-    <div class="a0-message a0-tip row">
-      <b><?php echo _e( 'Pro Tip' ); ?>:</b>
-      <?php echo _e( 'Already set up another WordPress instance with Auth0? ' ); ?>
-      <a href="admin.php?page=wpa0-import-settings"><?php echo _e( 'Click here' ); ?></a>
-      <?php echo _e( ' to save time and import that site\'s SSO settings.' ); ?>
-    </div>
-
   </div>
 </div>
-
-
-<script type="text/javascript">
```

Comment:
Removed the auto-setup here (was forcing a data migration which not all sites need) and moved the rest to `assets/js/admin.js`